bigflow/_version.py

```diff
@@ -1 +1 @@
-__version__ = '1.3.1'
```

Comment:
BTW. I suppose it is better to change any version as a separate commit with appropriate message (like "Bump version to XXX"). Also it would be nice to put a tag for such commit.

Reply:
Ok, I will create tag after creating new version is in pip